build.sbt

```
@@ -227,6 +244,8 @@ lazy val props = new {
 
   val MinitestVersion_2_11 = "2.8.2"
   val MinitestVersion = "2.9.6"
+
+  val MunitVersion = "0.7.26"
```

```suggestion
  val MunitVersion = "0.7.27"
```
or higher to solve `a linking error` from Scala.js build.